docs/source/developer_guides/method_comparison/lora.md

```diff
@@ -0,0 +1,196 @@
+# LoRA (Low-Rank Adaptation)
+
+## Overview
+LoRA is a parameter-efficient fine-tuning method that introduces trainable low-rank matrices into transformer layers. It's particularly effective for large language models and offers a good balance between performance and resource efficiency.
+
+## Key Features
+- Memory efficient (~0.1% of base model parameters)
+- Minimal impact on inference speed
+- Easy to implement and use
+- Compatible with most transformer architectures
```

Comment:
It's worth mentioning that LoRA is the most feature rich PEFT method, especially when it comes to supporting multiple quantization methods.